htdocs/compta/paiement_vat.php

```diff
@@ -277,7 +277,7 @@
 			} */
 			$remaintopay = $objp->amount - $sumpaid;
 			print '<input type=hidden class="sum_remain" name="'.$nameRemain.'" value="'.$remaintopay.'">';
-			print '<input type="text" class="right width100" name="'.$namef.'" id="'.$namef.'" value="'.$remaintopay.'">';
+			print '<input type="text" class="right width100" name="'.$namef.'" id="'.$namef.'" value="'.GETPOST('amount_'.$objp->id, 'intcomma').'">';
```

Comment:
The first time we entry the page, value will not be prefilled.
May be you should make here :
GETPOSTISSET($namef) ? GETPOST($namef, 'alpha') : remaintopay